website/content/api-docs/system/internal-ui-version.mdx

```diff
@@ -0,0 +1,49 @@
+---
+layout: api
+page_title: /sys/internal/ui/version - HTTP API
+description: >-
+  The `/sys/internal/ui/version` endpoint exposes the software version of Vault.
+---
+
+# `/sys/internal/ui/version`
+
+The `/sys/internal/ui/version` endpoint is used to expose the software version
+of Vault to the UI so that it can be displayed to logged in users.
+
+This is currently only being used internally for the UI. Due to the nature of its
+intended usage, there is no guarantee on backwards compatibility for this endpoint.
+
+## Get version
+
+This endpoint exposes the software version of Vault to the UI.
```

Comment:
```suggestion
Return the current software version of Vault.
```

Style correction: avoid "this" as a pronoun, technically the endpoint just returns information. The fact that we use it in the UI is secondary